cpp/src/parquet/page_index.h

```diff
@@ -25,14 +25,13 @@
 
 namespace parquet {
 
-class ColumnDescriptor;
-class EncodedStatistics;
-class FileMetaData;
+class Decryptor;
+class Encryptor;
 class InternalFileDecryptor;
+class InternalFileEncryptor;
```

Comment:
It would probably be useful to have a `encryption/type_fwd.h` to centralize all such forward declarations?

Reply:
Done. Please take a look again. Thanks!